readthedocs/templates/projects/project_advanced_settings_helptext.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<p class="empty">
+  {% blocktrans with config_file_link="https://docs.readthedocs.io/page/config-file" %}
```

Comment:
I'd link directly to the v2 config file docs: https://docs.readthedocs.io/page/config-file/v2.html

Reply:
Also, you might consider adding `trimmed` to the `blocktrans` tag.